pages/design/rule-template.md

```diff
@@ -58,4 +58,54 @@ Support for XXX is known to be limited in some assistive technologies. If any of
 - The WCAG 2.0 Techniques already contain examples and code snippets to illustrate which content passes or fails the test. Whenever possible auto-wcag refers to those. Another source for test cases is the W3C Before and After Demonstration.
 - Other references
 
+## Test Cases
+
+### Passed
+
+#### Pass example 1
+
+Briefly describe what passed in the html code below
+
+```html
+<!-- passing html code snippet -->
+```
+
+#### Pass example X
+
+... Add one test case for each "reason" an element can pass
+
+### Failed
+
+#### Failure example 1
+
+Briefly describe what failed in the html code below
+
+```html
+<!-- failing html code snippet -->
+```
+
+#### Failure example X
+
+... Add one test case for each "reason" an element can fail
+
+### Inapplicable
+
+#### Inapplicable example 1
+
+Briefly describe why the html code snippet is inapplicable
+
+```html
+<!-- inapplicable code snippet -->
 ```
+
+#### Inapplicable example X
+
+... Add one test case for each "reason" an element can be inapplicable
+
+````
+
+## Composed rules
```

Comment:
Is this section meant to go into all Composed rules? 

I would suggest that we create one page and link to that one instead, so that we don't risk having to update the wording in all composed rules if something is being changed.